test/integration/freetext_editor_spec.mjs

```diff
@@ -111,8 +111,9 @@ describe("FreeText Editor", () => {
           await waitForSelectedEditor(page, editorSelector);
           await waitForStorageEntries(page, 1);
 
-          const alert = await page.$eval("#viewer-alert", el => el.textContent);
```

Comment:
Note that, as a small bonus, we don't shadow the `alert` builtin anymore with this change.